src/ServiceStack/RequestLogsFeature.cs

```diff
@@ -96,7 +96,7 @@ public void Register(IAppHost appHost)
             }
 
             appHost.GetPlugin<MetadataFeature>()
-                .AddDebugLink(AtRestPath, "Request Logs");
+                .AddDebugLink(AtRestPath.TrimStart('/'), "Request Logs");
         }
```

Comment:
The issue should be fixed in `.AddDebugLink()` or when link is displayed, but not here. 
What's the url that gets displayed?